$NetBSD: patch-tar.cabal,v 1.2 2022/02/11 13:03:51 pho Exp $

Relax the dependency constraint. The package is known to be compatible
with GHC 9.

--- tar.cabal.orig	2001-09-09 01:46:40.000000000 +0000
+++ tar.cabal
@@ -40,7 +40,7 @@ flag old-bytestring
 
 library
 
-  build-depends: base       >= 4 && < 4.14,
+  build-depends: base       >= 4 && < 5,
                  filepath             < 1.5,
                  array                < 0.6,
                  containers >= 0.2 && < 0.7,
@@ -49,12 +49,12 @@ library
   if flag(old-time)
     build-depends: directory < 1.2, old-time < 1.2
   else
-    build-depends: directory >= 1.2 && < 1.4, time < 1.10
+    build-depends: directory >= 1.2 && < 1.4, time < 1.13
 
   if flag(old-bytestring)
     build-depends: bytestring-builder >= 0.10.4.0.2 && < 0.11, bytestring == 0.9.*
   else
-    build-depends: bytestring == 0.10.*
+    build-depends: bytestring >= 0.10 && < 0.12
 
   if !impl(ghc >= 8.0)
     build-depends: semigroups == 0.18.*
